tests/ttnn/unit_tests/operations/test_composite.py

```diff
@@ -97,10 +97,46 @@ def test_unary_composite_clamp_ttnn(input_shapes, device):
     in_data1, input_tensor1 = data_gen_with_range(input_shapes, -100, 100, device)
     min = -10
     max = 10
-    output_tensor = ttnn.clamp(input_tensor1, min, max)
+    output_tensor = ttnn.clamp(input_tensor1, min=min, max=max)
     golden_function = ttnn.get_golden_function(ttnn.clamp)
-    golden_tensor = golden_function(in_data1, min, max)
+    golden_tensor = golden_function(in_data1, min=min, max=max)
+
+    comp_pass = compare_pcc([output_tensor], [golden_tensor])
+    assert comp_pass
 
+
+@pytest.mark.parametrize(
```

Comment:
Ideally this should be a single test. Avoid such code duplications. 

In test input variations you can provide values for both min and max and in some cases they can be null, including both - to check that error is expected there.